magento/pwa-studio
# PWADevServer: Avoid binding/listening by domain name

## Description

Pls check related issue description

## Related Issue





Closes #2428

## Acceptance 



### Verification Stakeholders

### Specification
Affects  `pwa-buildpack` package.

### Verification Steps


1. Run current docker setup
2. Note correct https scheme
3. Web-site should work as usual.
4. You could try above steps with custom origin as well (even without docker).

## Screenshots / Screen Captures (if appropriate)
Example: https://prnt.sc/sm4047

## Checklist

* I have updated the documentation accordingly, if necessary.
[X] I have added tests to cover my changes, if necessary.
